tensorboard/meta.yaml

```diff
@@ -1,31 +1,30 @@
 {% set name = "tensorboard" %}
-{% set version = "2.6.0" %}
+{% set version = "2.8.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/py3/t/tensorboard/tensorboard-{{ version }}-py3-none-any.whl
-  sha256: f7dac4cdfb52d14c9e3f74585ce2aaf8e6203620a864e51faf84988b09f7bbdb
+  sha256: 65a338e4424e9079f2604923bdbe301792adce2ace1be68da6b3ddf005170def
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  skip: True  # [py<37]
```

Comment:
Should be `py<36`, see https://github.com/tensorflow/tensorboard/blob/2.8.0/tensorboard/pip_package/setup.py#L67